samples/client/petstore/java/src/main/java/io/swagger/client/model/ApiResponse.java

```diff
@@ -1,64 +0,0 @@
-package io.swagger.client.model;
```

Comment:
I regenerated petstore java sample by removing `samples/client/petstore/java/src/main` first, to remove the old files in different directory. Then the generation does not generate this `ApiResponse.java` file, is it still needed?